packages/toolkit/src/entities/models.ts

```diff
@@ -24,7 +24,7 @@ export type Dictionary<T, Id extends EntityId> = Partial<Record<Id, T>>
 /**
  * @public
  */
-export type DictionaryNum<T> = Dictionary<TemplateStringsArray, number>
```

Comment:
I think that with this PR, this type could be pretty much completely removed - or only be kept for backwards compat, but not really referenced any more/